packages/atlas/.env

```diff
@@ -9,10 +9,10 @@ VITE_DEVELOPMENT_NODE_URL=wss://atlas-dev.joystream.org/ws-rpc
 VITE_DEVELOPMENT_FAUCET_URL=https://atlas-dev.joystream.org/member-faucet/register
 VITE_DEVELOPMENT_OFFICIAL_JOYSTREAM_CHANNEL_ID=12
 
-VITE_NEXT_ORION_URL=https://atlas-next.joystream.org/orion/graphql
-VITE_NEXT_QUERY_NODE_SUBSCRIPTION_URL=wss://atlas-next.joystream.org/query-node/server/graphql
-VITE_NEXT_NODE_URL=wss://atlas-next.joystream.org/ws-rpc
-VITE_NEXT_FAUCET_URL=https://atlas-next.joystream.org/member-faucet/register
+VITE_NEXT_ORION_URL=https://34.229.168.20.nip.io/orion/graphql
+VITE_NEXT_QUERY_NODE_SUBSCRIPTION_URL=wss://34.229.168.20.nip.io/query-node/server/graphql
+VITE_NEXT_NODE_URL=wss://34.229.168.20.nip.io/ws-rpc
+VITE_NEXT_FAUCET_URL=https://34.229.168.20.nip.io/member-faucet/register
```

Comment:
You pushed too many changes, I guess.

Reply:
I certainly wanted to do this too fast 😓

Reply:
Fixed now!